LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/stylevalue-subclasses/cssScale.tentative-expected.txt

```diff
@@ -1,7 +1,7 @@
 
 PASS Constructing a CSSScale with an angle CSSUnitValue for the coordinates throws a TypeError
 PASS Constructing a CSSScale with a CSSMathValue that doesn't match <number> for the coordinates throws a TypeError
-PASS Constructing a CSSScale with an invalid division by px/px for the coordinates throws a TypeError
+FAIL Constructing a CSSScale with an invalid division by px/px for the coordinates throws a TypeError assert_throws_js: function "() => new CSSScale(coord, 0)" did not throw
```

Comment:
Same question

Reply:
This one was incorrectly checked in by a chrome engineer. See https://github.com/w3c/csswg-drafts/issues/10733